### ISSUE TITLE
Ash drakes are frankly way too fucking hard to kill

### ISSUE THREAD

**Issue by Borisvanmemes** [tgstation/tgstation#40576]
Not a bug but it is something that needs fixing, it's just, way too hard, everything about it, EVERYTHING.

**Comment by ma44** [comment]
I ded

**Comment by Barhandar** [comment]
Can confirm, it's utterly impossible to dodge and fight considering 1: it does damage to anything if any attack, including firespray, connects (so fireproofing is useless); 2: it sprays fire directly at you, so if you're in shooting range _on a server_ it's near-guaranteed hit unless you were moving sideways and there isn't a telegraph; 3: the instant swoop means, again, unless you were already moving out of the attack zone you get swooped; 4: the lava arena is far worse bullshit than old swoop because not only drake remains invulnerable, you also have to do pinpoint precise dodges.

Oh and every single attack freezes my client so it's _extra_ impossible to dodge.  
And they all are _explicitly designed_ to remove player agency (e.g. getting fireproofing for the fire) with the direct damage thing.

**Comment by zeroisthebiggay** [comment]
Before merge the drake had around three varieties of swoop. Normal swoop, fire snake, and triple swoop.
Now drake only has one _insanely fucking stupid_ swoop attack which consists of:
Swoop up and create lava patches everywhere, these patches have a very short frame before they _actually appear_ and also supposedly deal damage to anything ontop of them before even spawning from what i've been told, once these lava patches are created there's a very brief moment you have to get a clear path and not panic, because drake lands from the swoop **fucking instantly** and will crush you if you're not actively moving, guaranteeing your death due to the next attack, the triple fire spray. As stated above the drake's fire spray attack is no longer in a + pattern but instead in a trident shooting towards it's target, after a swoop it will spray fire in the trident pattern **three times consecutively**, dealing 60 damage (not including the afterburn) to anyone unlucky enough to get hit by all 3, and since if you fucked up and got stomped by the drake/stepped in the lava you're probably way too slow to get far away to properly dodge this attack. Now let's say you dodge this attack perfectly every time, the drake has gotten the new "arena attack" which seeing as how most _servers (not client run servers mind you)_ you'll probably overstep from the *one tile* that you're safe from the almost certain death lava, the drake also stomps down when this attack is finished as fast as he does when he swoops because fuck you.

TL;DR Drake is new Colosssus, no reason to fight him as his loot is still as good/bad as pre-merge, just go fight Bubbles at that point.

**Comment by sagattyy** [comment]
My only issue with the new drake is that he's impossible to fight if he manages to get inside the station

**Comment by Whoneedspacee** [comment]
I’m intending for all the megafauna to “be like colossus”, it’s frankly fucking retarded that a man with a PKA and no prep can kill a goddamn dragon and just get some op ash storm prevention gear in two seconds.

I toned down some attacks slightly and gave them very readable tells for what attack is happening. I’m not sure about your game freezing though, other people who have fought it seem to have no problem when I talked to them on discord.

If you get hit by all 3 of the flame breaths frankly I don’t feel bad for you and you fucked up. It’s no different than other megafauna attacks that can instantly destroy you if you’re in the wrong place at the wrong time. If you didn’t pack some decent healing like a medipen or a legion soul then that’s your bad.

The address that you “might as well kill bubblegum” is not going unheard, he will be buffed as well, probably harder than drake is if I do my job right. Easily killing a monster in a few minutes of the round starting for a projectile sword that cuts off limbs, or a one free dead person contract, is frankly stupid.

Megafauna should be the end game content of lavaland, build your gear up then fight them unless you want to risk dying early on.

**Comment by ishitbyabullet** [comment]
@Whoneedspacee No fucking shit it's hard, dude.

We didn't add the dragonborn job to ss13 yet. What the fuck were you expecting when we can't even use dragon shouts yet?

**Comment by Iamgoofball** [comment]
Have you tried fighting it with a friend?

**Comment by ishitbyabullet** [comment]
@Iamgoofball Serious post here:

It only attacks if it's attacked. While obviously both of the players would attack the drake, it's AI isn't smart enough to really alternate between the two at random from what I remember and experienced. They would still have the same issues but now they just have a buddy who may die in the same way as them if unlucky.

**Comment by ishitbyabullet** [comment]
@Barhandar 

> Other megafauna attacks don't specifically seek you out (and when they do they don't DO RIDICULOUS DAMAGE).

The few tracking attacks I can remember clearly are
- bubblegum charge
- hierophant attacks in general
- using sleeping carp against colossus so he chases you
- ash drake sweep before this PR
The only one that doesn't just fuck you over is hierophant lol

The man who made the pr is brave enough to even try balancing it. The alternative of 'op bosses that no one can fight' is 'easy bosses that powergamers kill in ten minutes.' Unless it's so bad of an idea (_like not adding the dragonborn job ffs_) then it doesn't mean he is **THE CSGO OF GAME DESIGN** or some shit

Yet again, I don't really try attacking the ash drake and haven't done so for a long time, so maybe my mind would change if I did the actual fight. I feared that the lava transformation attack would be too OP and if it works on station/places a crap ton of lava spots in ways that are way too brutal and unforgiving if you even lag for five seconds then that's just not fair. Not sure, though.

**Comment by Barhandar** [comment]
>The few tracking attacks I can remember clearly are

I meant attacks that do damage every step like the firebreath, i.e. only hiero. Amended previous comment.  
Also pretty sure bubblegum doesn't track you inasmuch as it charges at your previous position, hence the ease of the fight.
>The alternative of 'op bosses that no one can fight' is 'easy bosses that powergamers kill in ten minutes.'

You're completely ignoring "bosses that actually have reasonable and fitting difficulty" option. Which _is_ an option, even though appeal to powergamers is a fallacy because powergamers will be able to kill this drake with no preparation (BECAUSE PREPARATION IS USELESS LOL GG) as well, it'll just be _fewer_ powergamers. And fewer people in general being able to access the content behind the drake.

Is this a good thing? That's subjective, but in my opinion drake armour isn't special enough to justify an impossible boss.
>if you even lag for five seconds

If you lag for five seconds with this drake, you're dead.
> then it doesn't mean he is THE CSGO OF GAME DESIGN

Comparison to CSGO was about twitch dodges (having minimal time to react to something and hence doing it on automatic control) being used instead of reaction dodges (having enough time to react to something to do it consciously).

**Comment by bgobandit** [comment]
If they're the end-game content of Lavaland then they shouldn't spawn directly near the mining camp. Some miners actually want to mine for ore without being instantly accosted by the equivalent of a final boss at level 1

**Comment by Magikarpet** [comment]
tl;dr below if you dont wanna read it 

Terrible change purely because of how laggy the servers are, it feels like I'm gambling half the time with where I'll be while fighting megafauna and while I can compensate for every other megafauna mob easily with simple playstyle changes in being more defensive, it feels dreadful compensating for the drake due to how fast the fire breath is and how instant its swoops are.

The point around it being like the colossus is horridly wrong as the colossus has very obvious patterns you can read from fighting it, the blasts almost always have massive holes you can easily observe to dodge or prepare for based on the previous attack while the tracking trident combines the speed and aggressiveness of bubblegum with the aoe radius and crowd controlling(Yes, a million bullets being shot out is crowd control as you can't just run away)of a colossus. Tracking should probably only work at your current position while the fire breath starts rather than expanding out and tracking you, though i may be wrong on that last point if it was just the server being so laggy that it felt like the fire was constantly chasing me in erratic and weird patterns.

I'll deal with it and try to learn how to fight it again later when the servers are less laggy but right now it just feels like an attempt to turn the drake into the colossus while not understanding the colossus. The colossus is a very methodical fight of understanding its attack patterns and dodging them when you see them, or more ideally anticipating the patterns based on its last attack cycle. It's slow for a reason, while it can pin you down with bullets it's very easy to run away or dodge it flat out. The current drake feels more like a tracker heavy aggression boss, like bubblegum but without any mechanic to limit its aggression on you(Avoid standing on or around blood, go to a new clean area to reduce its teleports and keep a healthy running pace away from it). 

A few changes that I think it requires before it could be considered complete is for swooping to be slower while maybe scaling up in speed with health loss, the fire breath having a greater delay in its bursts which shortens as you lower its health, the fire pattern being wider and more generous as the first 2-3 tiles of the boss end up being fire 90% of the time due to how it balloons, or for the repeated fire blasts to come at different hp intervals being 1 until 67%, 2 until 34%, and below that being it triple fire blasting. I think these would help in making the drake not feel like a ratty boss that just spams damage particles everywhere mindlessly that you dodge on impulse and not out of strategy. 

if it's too much to read for you tl;dr paragraph spaced out: 

Its aggression feels too patternless and wild, almost like a combination colossus and bubblegum but without the weaknesses of either while having their strengths.
 
Changes that would be good to make it saner would be a speed reduction in the intervals between blasts during its triple fire blasts which decreases to potentially being even faster than it is now as it loses hp to allow for more mobility, or a reduction in how many times it breathes fire until it hits certain hp thresholds unlocking more blasts(100-67%, 67-34%, 34%-0%). 

With the lag of the server it feels too unreliable and like fighting the drake is gambling rather than using skill fighting it, while fighting the colossus or bubblegum while lagging is much easier due to their weaknesses with mobility or range which you can compensate for to counter them during laggy peak hours.

**Comment by Repukan** [comment]
It doesn't need to be easy.

If you want end game content, then practice or prepare yourself for it.

This is all pissbabies mad they can't get easy Ash proof high resist armor with no effort anymore

**Comment by Repukan** [comment]
Furthermore the megafauna fights are OPTIONAL.

Want Ash proof clothing? Collaborate with xeno or make a mech. What's that, you want robust armor with it too? Well maybe work with cargo to order swat suits.

There's no obligation for you to fight megafauna. It's not included in bounties, it's not tied to game mode mechanics, nor is it imperative to end rounds. You just want easily accessible powerful loot.

**Comment by ChangelingRain** [comment]
Its attacks should definitely be slower, to account for latency and general server load.

**Comment by Whoneedspacee** [comment]
You act like prep doesn’t do anything, having a firefighter ripley versus any mech is actually a boon to fighting drake whereas before any mech 2 seconds into the round could cheese drake.

Also, the triple fire blast ALWAYS happens at the same exact time after he lands during the lava swoop. People I’ve talked to on the discord have had no trouble dodging this by simply walking the fuck away.

Colossus has no attack patterns, his attack pattern is knowing the default cooldown time on his attacks and punishing him in between that. I meant make the bosses like colossus in difficulty, not how you fight them. All megafauna should strive to be something that can just compound destroying you if you’re unprepared for that.

Not sure what you mean by megafauna attacks not seeking you out, colossus shotgun definitely seeks me out, and it definitely insta fucks me, there’s also a triplet variant.

**Comment by AlexTheSysop42** [comment]
@barhendar
t. person who's never beat heirophant
3-4 hits from the snake is basically marking you for certain death if you don't have a legion core because any slowdown = dead
it also goes hilariously fast in the later phases and is basically an instakill if you stand still for more than a second while it's there (and it lasts long enough that it almost always is)

**Comment by Tlaltecuhtli** [comment]
maybe get geared before fighting endgame bosses? have you ever played an rpg before? this is NOT call of dooty

**Comment by HydroFloric** [comment]
@Whoneedspacee I don't disagree with what your saying just want to correct you. Since a new pr by imsxz all plasma cutter blasts were Nerfed including mechs. The mech only does 10 damage now so its basically worthless for fighting a megafauna.

**Comment by bgobandit** [comment]
RPGs don't put the final boss in the first fucking room of the game though*

* yes I know about the barrel

**Comment by HydroFloric** [comment]
@bgobandit Its not like megafauna spawning near the base is a new thing. You can have 2 fucking colossus spawn 50 meters away from the base if you are unlucky.

**Comment by Magikarpet** [comment]
not saying that it should be easy but as it is it involves less strategy than you might imagine repukan

To add, endgame on the servers is 10-25 minutes into the round and that's what almost everything new has been balanced around. Most rounds don't hit the 60 minute mark to do your mythical virus+5 legion core+gatfruit+heavy armor to cheese bosses and not learn their actual attack and movement patterns  (Xenobiology consoles, new rnd, botany, assmos, and a few other features I forgot like a dummy are scaled around completing 80-90% of a job by 20-25 minutes max as examples of changes to an existing role to make it faster in a lot of ways) 

But the real meme is that the bubblegum "buff" made it 10x easier to kill bubblegum while the drake one makes it miserable to fight against 

The thing that genuinely is aggrivating is that even around lowpop the server lags, feeling like you're playing with 150 ping on another game or 200+ ping on highpop meaning that you have to act 

regardless i don't kill megafauna for the loot i like killing them for the challenge, though the change feels like less of a challenge and more like "Let's buff the drake by making it spam abilities and giving it a massive speed boost" but as it is the server load renders it hell as almost every round slows to a crawl with lag in my experience so that may be the bigger issue here

**Comment by kingoftheautism** [comment]
i ded

**Comment by Whoneedspacee** [comment]
The bubblegum buff was a lazy change by Kor before he left to be honest, I’ll try to rework it into a proper melee boss but it does take time.
